storage_service/src/storage_service/storage.py

```diff
@@ -0,0 +1,27 @@
+"""
+Provides storage instruments for synchronization state and for arbitrary data
+"""
+
+
+class DataBase(object):
+    """
+    Making the class attributes global
+    """
+
+    _shared_data = {}
+
+    def __init__(self):
+        self.__dict__ = self._shared_data
+
+
+class Data(DataBase):
```

Comment:
What is the purpose of it?

Reply:
At the very beginning we're going to store list of onedrive items in a pickled singleton. Over time, we will replace it with the database if necessary.